src/index.html

```diff
@@ -22,6 +22,408 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="header">
```

Comment:
The tests are failing because you used `div` instead of `header` tag
```suggestion
    <header class="header">
```